src/components/calcite-notice/calcite-notice.tsx

```diff
@@ -139,13 +145,13 @@ export class CalciteNotice {
   /** close the notice emit the `calciteNoticeClose` event - <calcite-notice> listens for this */
   @Method() async close() {
     this.active = false;
-    this.calciteNoticeClose.emit({ requestedNotice: this.noticeId });
+    this.calciteNoticeClose.emit(this.el);
   }
 
   /** open the notice and emit the `calciteNoticeOpen` event - <calcite-notice> listens for this  */
   @Method() async open() {
     this.active = true;
-    this.calciteNoticeOpen.emit({ requestedNotice: this.noticeId });
+    this.calciteNoticeOpen.emit(this.el);
```

Comment:
Is this element passed through already as `event.target`? If so we could just use that instead of passing it through as `event.detail`

Reply:
@paulcpederson Updated. How's that look?

Reply:
✨ 🚢 ✨

Reply:
Yew!